src/Makefile

```diff
@@ -15,4 +16,4 @@ requirements: requirements.in
 upgrade:
 	pur --patch=Django -r requirements.in
 	pur --no-recursive -r requirements_dev.txt
-	pip-compile -v --upgrade --output-file requirements.txt requirements.in
```

Comment:
This means we're not doing non-pinned package upgrades inside `make upgrade` anymore?

Reply:
Hmm, good point. I think this change was a mistake. I initially thought that it would disregard the `--patch=Django` argument to `pur`. Eg `pur` would only patch-upgrade Django, but then the `--upgrade` argument to `pip-compile` would upgrade it to another major version (if available).

Anyway tested it out and it seems that `pip-compile` treats `requirements.in` and a constraint in the presence of `--upgrade`. So it is safe after all. Will revert.